crates/biome_analyze/CONTRIBUTING.md

```diff
@@ -782,7 +891,7 @@ and if you've done everything correctly, you should see some snapshots emitted w
 
 Check our main [contribution document](https://github.com/biomejs/biome/blob/main/CONTRIBUTING.md#testing) to know how to deal with the snapshot tests.
 
-### Documenting the rule
+### Documenting the Rule
```

Comment:
Is there a particular reason for this being upper case? It's not a proper name

Reply:
It's a heading, which typically is in "title casing"

Reply:
I honestly don't know what you're referring to

Reply:
See: https://en.wikipedia.org/wiki/Title_case

The rules vary depending on which style guide you reference, but generally it's:
- capitalize all words
   - except articles and conjunctions (eg. `the`, `a`, `but`, etc)
- Always capitalize the first word

Reply:
TIL :)